src/opencensus_influxdb.erl

```diff
@@ -38,6 +38,7 @@ normalise(<<",", Rest/binary>>, Result) -> normalise(Rest, <<Result/binary, "\\,
 normalise(<<" ", Rest/binary>>, Result) -> normalise(Rest, <<Result/binary, "\\ ">>);
 normalise(<<"=", Rest/binary>>, Result) -> normalise(Rest, <<Result/binary, "\\=">>);
 normalise(<<"\"", Rest/binary>>, Result) -> normalise(Rest, <<Result/binary, "\\\"">>);
+normalise(<<"/", Rest/binary>>, Result) -> normalise(Rest, <<Result/binary, "_">>);
```

Comment:
out of curiosity, why replace by an underscore rather than escaping like with all other special characters?

Reply:
OpenCensus use `/` as a general purpose separator in metrics and InfluxDB for that purpose use `_`. That was the main reasoning behind using `_` there.

Reply:
understood. Might make sense to put it in a comment